hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/AbstractRootedOzoneFileSystemTest.java

```diff
@@ -2610,4 +2611,35 @@ private void deleteFiles(Path base, List<String> fileNames) throws IOException {
     }
   }
 
+  @Test
+  public void testSetReplication() throws IOException {
```

Comment:
Would it be possible to implement (most of) the new test in `OzoneFileSystemTests` and reuse it in `AbstractOzoneFileSystemTest` and `AbstractRootOzoneFileSystemTest`?

Reply:
Done.